pkg/types/baremetal/defaults/platform.go

```diff
@@ -51,4 +52,15 @@ func SetPlatformDefaults(p *baremetal.Platform, c *types.InstallConfig) {
 			p.ApiVIP = vip[0]
 		}
 	}
+
+	if p.IngressVIP == IngressVIP {
+		// This name should resolve to exactly one address
+		vip, err := net.LookupHost("test.apps." + c.ClusterDomain())
```

Comment:
Is this a safe assumption for the ingress DNS entry?

Reply:
@celebdor or @yboaron can you comment on this - I think this should work and always resolve to the ingress VIP, right?

Reply:
@shardy - that's correct.
The Ingress VIP should be defined as wildcard dns record (*.apps.$ClusterDomain),
so any prefix added to  '.apps.$ClusterDomain' should resolve to the ingress VIP.

Reply:
I guess my main concern was whether the .apps. part was always the case. Is the prefix hard-coded or can users configure that?

Reply:
@cyberton,  Hmm, that's a good point, I was 100% sure that it's hardcoded value, so according to OCP4.1 documentation [1], seems that it should be hardcoded value for baremetal UPI.  

To double check that I sent a Q to it OCP slack channel [2]

[1] https://docs.openshift.com/container-platform/4.1/installing/installing_bare_metal/installing-bare-metal.html#installation-dns-user-infra_installing-bare-metal 
[2] https://coreos.slack.com/archives/C68TNFWA2/p1561533033492300

Reply:
@cybertron , it should be safe to use '.apps.$ClusterDomain', see [1]
[1] https://coreos.slack.com/messages/CCH60A77E/convo/CCH60A77E-1561545440.277400/